Error: VM SATA Disk Passthrough

It does create the VM and external disk is attached, however terraform crashes and does not store the vm on state file. Tried to import the VM later and same issue on terraform plan.

```hcl
resource "proxmox_vm_qemu" "main" {
  target_node = "pve"

  disks {
    sata {
      sata0 {
        passthrough {
          file = "/dev/disk/by-id/wwn-0x5000c5009e0827fa-part1"
        }
      }
    }
  }
}
```

 Error: Plugin did not respond
│ 
│   with proxmox_vm_qemu.main,
│   on test.tf line 1, in resource "proxmox_vm_qemu" "main":
│    1: resource "proxmox_vm_qemu" "main" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may
│ contain more details.
╵

Stack trace from the terraform-provider-proxmox_v3.0.1-rc6 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0xa8 pc=0x1014215c4]

goroutine 71 [running]:
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk.terraform_Disks_QemuSataStorage(0x14000a102c0, 0x140003c8280?)
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk/terraform_disks.go:142 +0x744
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk.terraform_Disks_QemuSataDisks(0x14000a99b90, 0x10165df33?)
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk/terraform_disks.go:102 +0x40
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk.terraform_Disks_QemuDisks({0x0?, 0x14000a99b90?, 0x0?, 0x0?}, 0x14000a98210?)
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk/terraform_disks.go:28 +0x5c
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk.Terraform_Unsafe(0x140008c3f88?, 0x14000a10220, 0x0?)
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/qemu/disk/terraform.go:13 +0x98
github.com/Telmate/terraform-provider-proxmox/v2/proxmox.resourceVmQemuRead({0x1400011a2d0?, 0x0?}, 0x14000305600, {0x1017e7740?, 0x1400024e780})
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/resource_vm_qemu.go:1224 +0x524
github.com/Telmate/terraform-provider-proxmox/v2/proxmox.resourceVmQemuCreate({0x101953760, 0x14000162b60}, 0x14000305600, {0x1017e7740?, 0x1400024e780})
        github.com/Telmate/terraform-provider-proxmox/v2/proxmox/resource_vm_qemu.go:914 +0x26ec
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0x140000eaa80, {0x1019536b8, 0x14000a80d50}, 0xd?, {0x1017e7740, 0x1400024e780})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:806 +0xe8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0x140000eaa80, {0x1019536b8, 0x14000a80d50}, 0x14000afe270, 0x1400075e980, {0x1017e7740, 0x1400024e780})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:937 +0x86c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0x14000398000, {0x1019536b8?, 0x14000a80c60?}, 0x140007fe500)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/grpc_provider.go:1153 +0xb08
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0x140002ae000, {0x1019536b8?, 0x14000a80240?}, 0x14000162070)
        github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:865 +0x2b0
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x101920220?, 0x140002ae000}, {0x1019536b8, 0x14000a80240}, 0x1400075e180, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:518 +0x164
google.golang.org/grpc.(*Server).processUnaryRPC(0x140001b3200, {0x1019536b8, 0x14000a801b0}, {0x101957638, 0x140002aa480}, 0x14000a84000, 0x140005351a0, 0x101ea3018, 0x0)
        google.golang.org/grpc@v1.63.2/server.go:1369 +0xba0
google.golang.org/grpc.(*Server).handleStream(0x140001b3200, {0x101957638, 0x140002aa480}, 0x14000a84000)
        google.golang.org/grpc@v1.63.2/server.go:1780 +0xc80
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 49
        google.golang.org/grpc@v1.63.2/server.go:1030 +0x150

Error: The terraform-provider-proxmox_v3.0.1-rc6 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

--- a/proxmox/Internal/resource/guest/qemu/disk/terraform_disks.go
+++ b/proxmox/Internal/resource/guest/qemu/disk/terraform_disks.go
@@ -139,7 +139,7 @@ func terraform_Disks_QemuSataStorage(config *pveAPI.QemuSataStorage, ciDisk *boo
 			schemaFile:       config.Passthrough.File,
 			schemaReplicate:  config.Passthrough.Replicate,
 			schemaSerial:     string(config.Passthrough.Serial),
-			schemaSize:       convert_KibibytesToString(int64(config.Disk.SizeInKibibytes)),
+			schemaSize:       convert_KibibytesToString(int64(config.Passthrough.SizeInKibibytes)),
 		}
 		terraformQemuDiskBandwidth(mapParams, config.Passthrough.Bandwidth)
 		return []interface{}{map[string]interface{}{

